fog/sample-paykit/src/client.rs

```diff
@@ -779,6 +792,7 @@ mod test_build_transaction_helper {
             &recipient_account_key.default_subaddress(),
             super::BlockIndex::max_value(),
             fake_acct_resolver,
+            true,
```

Comment:
nit: on the list ingress keys PR, Eran and I were talking about how bool function params generally decrease readability. One way to solve this is to define an enum that has two values "USE_RTH_MEMOS" or "USE_DEFAULT_MEMOS." Another option is to inline comment the param nam "/* use_rth_memos= */. 

Might be worth implementing, but don't feel strongly.